add indices to FAQ pages

zimmy87 opened the pull request:
<!-- Thank you for submitting a pull request! -->
<!-- ⚠️⚠️ Do Not Delete This! pull_request_template ⚠️⚠️ -->
<!-- Please read our contribution guidelines: https://microsoft.github.io/AirSim/CONTRIBUTING/ -->

Fixes: #    <!-- add this line for each issue your PR solves. -->
<!-- Fixes: # -->
<!-- Fixes: # -->

## About
<!-- Describe what your PR is about. -->
This moves the FAQs for building on linux and windows to one central FAQ location for building that is accessible from the side pane.

## How Has This Been Tested?
<!-- Please, describe how you have tested your changes to help us incorporate them. -->
Tested via both GitHub preview and a locally run `mkdocs serve` command.

## Screenshots (if appropriate):

zimmy87 committed:
add indices to FAQ pages

this moves the FAQs for building on linux and windows to one central FAQ location for building that is accessible from the side pane

jonyMarino reviewed (changes requested):
@zimmy87 just one comment

zimmy87 committed:
Update faq.md

add link to package_binaries.md from main faq and fix links for both unreal environments

zimmy87 committed:
Update faq.md

changing list of "others" to a bullet-point list for better readibility

zimmy87 committed:
Update build_faq.md

add text of docs/package_binaries.md to "Other" section

zimmy87 commented on docs/package_binaries.md:
```diff
@@ -0,0 +1,6 @@
+# Packaging a binary including the AirSim plugin 
```
resolved with latest commits

zimmy87 committed:
Delete package_binaries.md

remove this file since its text is now part of build_faq.md

zimmy87 committed:
Update mkdocs.yml

remove link to package_binaries.md since it has been removed

zimmy87 commented:
all CR feedback has been addressed, moving ahead with merging
